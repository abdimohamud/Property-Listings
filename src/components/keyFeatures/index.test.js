import * as React from 'react'
import renderer from 'react-test-renderer'

import KeyFeatures from '.'

test('renders correctly', () => {
  const features = ['first', 'second', 'third']
  const tree = renderer.create(<KeyFeatures features={features} />).toJSON()
  expect(tree).toMatchSnapshot()
})
