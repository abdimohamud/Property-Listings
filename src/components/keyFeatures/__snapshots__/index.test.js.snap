// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <p
    className="text-bold"
  >
    Key Features
  </p>,
  <ul
    className="mt-0 mb-0"
  >
    <li>
      <small>
        first
      </small>
    </li>
    <li>
      <small>
        second
      </small>
    </li>
    <li>
      <small>
        third
      </small>
    </li>
  </ul>,
]
`;
